app/upgrades.go

```diff
@@ -60,6 +60,7 @@ var upgradesList = []string{
 	"3.0.8",
 	"v3.0.9",
 	"v3.1.0",
```

Comment:
shall we remove this?

Reply:
@yzang2019 yea I left it in because it shouldn't cause any issue. But I can just go ahead and remove

Reply:
Updated